shared/profile/folders/container.tsx

```diff
@@ -1,115 +1,10 @@
-            )
-            .map(tlfName => ({
-              isPublic: true,
-              isSelf: false,
-              openInFilesTab: () =>
-                dispatchProps._openInFilesTab(FsTypes.stringToPath(`/keybase/public/${tlfName}`)),
-              text: `public/${tlfName}`,
-            })),
-          // self private folder
-          {
-            isPublic: false,
-            isSelf: true,
-            openInFilesTab: () =>
-              dispatchProps._openInFilesTab(
-                FsTypes.stringToPath(`/keybase/private/${stateProps._ownUsername}`)
-              ),
-            text: `private/${stateProps._ownUsername}`,
-          },
-          // all other favorited private folders
-          ...stateProps._tlfs.private
-            .keySeq()
-            .filter(tlfName => tlfName !== stateProps._ownUsername)
-            .map(tlfName => ({
-              isPublic: false,
-              isSelf: false,
-              openInFilesTab: () =>
-                dispatchProps._openInFilesTab(FsTypes.stringToPath(`/keybase/private/${tlfName}`)),
-              text: `private/${tlfName}`,
-            })),
-        ]
-      : [
-          // this's profile's public folder
-          {
-            isPublic: true,
-            isSelf: true,
-            openInFilesTab: () =>
-              dispatchProps._openInFilesTab(FsTypes.stringToPath(`/keybase/public/${profileUsername}`)),
-            text: `public/${profileUsername}`,
-          },
-          // other favorited public folders where the profile user is a member
-          ...stateProps._tlfs.public
-            .keySeq()
-            .filter(tlfName => tlfName !== profileUsername && tlfName.split(/[,#]/).includes(profileUsername))
-            .map(tlfName => ({
-              isPublic: true,
-              isSelf: false,
-              openInFilesTab: () =>
-                dispatchProps._openInFilesTab(FsTypes.stringToPath(`/keybase/public/${tlfName}`)),
-              text: `public/${tlfName}`,
-            })),
-          // all favorited private folders where the profile user is a member
-          ...stateProps._tlfs.private
-            .keySeq()
-            .filter(tlfName => tlfName.split(/[,#]/).includes(profileUsername))
-            .map(tlfName => ({
-              isPublic: false,
-              isSelf: false,
-              openInFilesTab: () =>
-                dispatchProps._openInFilesTab(FsTypes.stringToPath(`/keybase/private/${tlfName}`)),
-              text: `private/${tlfName}`,
-            })),
-        ],
-})
-const hasFolders = Container.connect(mapStateToProps, mapDispatchToProps, mergeProps)(Folders)
-
-// @ts-ignore
-const noFolders: typeof hasFolders = Container.connect(
+const noFolders = Container.connect(
   () => ({}),
   () => ({}),
   () => ({loadTlfs: () => {}, tlfs: []})
 )(Folders)
 
-export default flags.foldersInProfileTab ? hasFolders : noFolders
```

Comment:
huh has this feature just been disabled forever? I don't remember it

Reply:
its false on all platforms